org.eclipse.tm4e.ui/src/main/java/org/eclipse/tm4e/ui/utils/PreferenceUtils.java

```diff
@@ -0,0 +1,42 @@
+package org.eclipse.tm4e.ui.utils;
+
+import org.eclipse.core.runtime.preferences.IEclipsePreferences;
+import org.eclipse.core.runtime.preferences.InstanceScope;
+import org.eclipse.tm4e.ui.internal.preferences.PreferenceConstants;
+
+public class PreferenceUtils {

```

Comment:
We should probably have a `getTM4EPreferenceStore()`

Reply:
Ok, I'll add it.